### ISSUE TITLE
Expose more configuration options on the command line

### ISSUE THREAD

**Issue by mario-grgic** [sql-formatter-org/sql-formatter#701]
**Describe the Feature**
It would be good to get more config options exposed on the command line. At the very least `--case` option to control keyword, function and data type case with values `upper`, `lower`.  This would allow one to use the command line without having to write multiple config files.

**Why do you want this feature?**
Usability.

**Comment by nene** [comment]
I think it's a fairly sensible request.

I see a couple of possible ways to implement it:

#### Separate command-line option for each config option

```
sql-formatter --keyword-case=upper --tab-width=4
```

The benefit is that the config options would be in the most natural form for command-line usage.

The downside is a lot of new config options (more complex command-line API) and a lot of additional code to maintain.

#### Some generic command line option to specify any config option

```
sql-formatter --define keywordCase=upper --define tabWidth=4
```

This sort of solution would keep the API pretty simple. But it still requires a good chunk of code to deal with each option. Like it needs to know that `4` needs to be converted to a number, while `upper` has to remain a string.

#### A way to write the JSON config inline

```
sql-formatter --config '{"keywordCase": "upper", "tabWidth": 4 }'
```

This would be by far the simplest to implement and maintain. Though on the downside it's the worst to type out on the command line. But on the plus side there's no translation needed to convert config in JSON file to config on command line.

I personally like the last approach the best.

**Comment by nene** [comment]
Implemented in 15.1.0 release.

**Comment by mario-grgic** [comment]
Thank you for being so responsive and implementing this so quickly. Awesome work, and I like the approach taken as well. This is very flexible now.
